src/supa/connection/fsm.py

```diff
@@ -44,6 +54,12 @@ class ReservationStateMachine(StateMachine):
     .. image:: /images/ReservationStateMachine.png
     """
 
+    log: BoundLogger
```

Comment:
Perhaps extract a SuPAStateMachine class with a logger property and a log function and use that as a Superclass for all other statemachine classes.

Reply:
```python
class SuPAStateMachine:
    logger: BoundLogger
    
    def log(self) -> None:
        self.logger.info("State transition", to_state=self.current_state.identifier, connection_id=str(self.model.connection_id))

```

Reply:
Good suggestion, I simplified the logging, see 30ac8d76be52c07e43d38bc5df39d606ae493f9e.